### PR TITLE
ci: update GitHub action

### PR DESCRIPTION
**Pull request by targos**
- Use actions/checkout@v2
- Upload coverage report now that Codecov supports tokenless uploads

### CODE REVIEW

**Comment by targos** [comment]
image

**Comment by targos** [comment]
No coverage report found. I must have missed something

**Comment by richardlau** [comment]
> 
> 
> No coverage report found. I must have missed something

The Travis job runs
https://github.com/nodejs/citgm/blob/18a041d861b55b86763f048c68b919e8ee1d3b34/.travis.yml#L8
which is
https://github.com/nodejs/citgm/blob/18a041d861b55b86763f048c68b919e8ee1d3b34/package.json#L21
while this PR is running
https://github.com/nodejs/citgm/blob/18a041d861b55b86763f048c68b919e8ee1d3b34/package.json#L19

so probably it's the `--coverage-report=lcov"`.

File: .github/workflows/nodejs.yml [hunk 1]
```diff
@@ -30,15 +30,17 @@ jobs:
...
+      - uses: actions/checkout@v2
+      - name: Use Node.js ${{ matrix.node-version }}
+        uses: actions/setup-node@v1
+        with:
+          node-version: ${{ matrix.node-version }}
+      - name: npm install and test
+        shell: pwsh
+        run: |
+          npm install
+          npm run tap
```
Comment (targos) [inline .github/workflows/nodejs.yml]:
```suggestion
          npm run tap -- --coverage-report=lcov
```

**Comment by targos** [comment]
@richardlau of course, I forgot. Thanks!

**Comment by targos** [comment]
Proof that the upload worked: https://codecov.io/gh/nodejs/citgm/commit/11651d781646a39c295faf9af3d2cae5f249b300/build

**Pull request merged**